Home work. Week one

no0dly opened the pull request:
### Домашнее задание номер один.

- добавлены компонены страниц(map, login, registration, profile);
- добавлен не стилизовынный хедер с кнопками навигации по страницам;
- добавлена навигация при помощи стейта;
- добавлены формы для страницы логина и регистрации;
- к формам прикручены фуки, состояние хранится в локальном стейте;

## Скриншот
![test](https://user-images.githubusercontent.com/8776095/92684737-072f5400-f2eb-11ea-9f28-6727d7f26069.gif)

no0dly committed:
feat: 123

432

no0dly committed:
feat: clean up

removed initial styles and layout

no0dly committed:
feat: added pages components, header

Added page components, added header with page change

no0dly committed:
feat: added login form

Added login form

no0dly committed:
feat: registration page form

Added form for registration page(unstyled)

Danil00524 commented on src/Header.js:
```diff
@@ -0,0 +1,43 @@
+                onClick={onPageChange}
+              >
+                Войти
+              </button>
```
Этот список лучше отобразить через map. Тут много одинакового кода и если нужно будет поменять тот же класс, то сделать это нужно будет во всех местах, вместо 1-го.

PS. Создаешь массив с объектами и как написал выше итерируешь его через map.

no0dly replied:
Поправил.

Danil00524 commented on src/Login.js:
```diff
@@ -0,0 +1,76 @@
+    password: "",
+  });
+
+  const onChangeHandler = (e) => {
```
onChange будет достаточно. Можно еще было бы указать, что именно, но компонент не такой большой.

no0dly replied:
Поправил

no0dly committed:
refactor(header, login page): minor review changes

Changed name for event handler onCnangeHandler to onChange, added array with mapping for navigation
buttons
